components/esp_http_server/src/httpd_main.c

```diff
@@ -39,7 +39,7 @@ static esp_err_t httpd_accept_conn(struct httpd_data *hd, int listen_fd)
              * therefore httpd_accept_conn() will be called again, but this time
              * with space available for one session
              */
-       }
+        }
```

Comment:
Some stray change?

Reply:
@shahpiyushv This is formatted by CLion and the `astyle` script provided by ESP-IDF.